app/views/application_mailer/sample_email.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Hi <%= @user.email %></h1>
+    <p>
+      Sample mail sent using smtp.
+    </p>
+    <% @user.posts.each do |user_post| %>
```

Comment:
Are you sending all the posts from the user? This email should have the pending tasks, fix this logic